Sources/MySQL/Connection.swift

```diff
@@ -162,6 +162,15 @@ public final class Connection {
         }
         return String(cString: error)
     }
-    
+
+    /**
+        Contains the last error code generated
+        by this MySQLS connection.
+    */
+    public var errorCode: UInt32 {
```

Comment:
Sounds good. It should match the `error` property, maybe both should be combined into one that returns both error message and code in an `Error` object.